pkg/maps/act/act.go

```diff
@@ -14,20 +14,24 @@ import (
 	"github.com/cilium/cilium/pkg/byteorder"
 	"github.com/cilium/cilium/pkg/datapath/linux/config/defines"
 	"github.com/cilium/cilium/pkg/ebpf"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
 	"github.com/cilium/cilium/pkg/option"
 	"github.com/cilium/cilium/pkg/service"
 )
 
 const ActiveConnectionTrackingMapName = "cilium_lb_act"
 
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "act-bpf")
```

Comment:
Consider instead taking and wiring through an `slog.Logger` - we're (slowly) moving away from package global log.